src/components/dataTable/utils.js

```diff
@@ -17,6 +17,19 @@ export const getDistinctFieldValue = (response, field) => {
       value && value.trim()
     ));
 };
+export const getIdentifier = (subject) => {
+  if(!subject) return;
+  return subject.curie?subject.curie:(subject.modEntityId?subject.modEntityId:subject.modInternalId);
+}
+
+export const getGeneObject = (subject) => {
+  if(!subject) return;
+  const gene = {
+    curie: getIdentifier(subject),
```

Comment:
I am not sure about the usefulness of this method which only returns a slimmed down version of a gene object, just curie and symbol.
I would rather replace 'gene.curie' occurrences with
getIdentifier(gene)

Reply:
Makes sense, will remove the method